corehq/util/test_utils.py

```diff
@@ -54,3 +55,17 @@ def get_json(cls, name, override_path=None):
     @classmethod
     def get_xml(cls, name, override_path=None):
         return cls.get_file(name, 'xml', override_path)
+
+
+def flag_enabled(toggle_class):
+    """
+    Decorate test methods with this to mock the lookup
+
+        @flag_enabled(toggles.MULTIPLE_LOCATIONS_PER_USER)
+        def test_something_fancy(self):
+            something.which_depends(on.MULTIPLE_LOCATIONS_PER_USER)
+    """
+    return mock.patch(
+        '.'.join([toggle_class.__module__, toggle_class.__class__.__name__, 'enabled']),
+        new=lambda *args: True,
+    )
```

Comment:
Interesting. I'd never seen that pattern before (or used the `new=` param), but makes sense after you explained it. Not sure how to mock the enabled method without `@patch` adding a new object into the method.